hedera-node/hedera-smart-contract-service-impl/src/main/java/com/hedera/node/app/service/contract/impl/exec/scope/HandleHederaNativeOperations.java

```diff
@@ -58,51 +58,33 @@ public HandleHederaNativeOperations(@NonNull final HandleContext context, @Nulla
         this.maybeEthSenderKey = maybeEthSenderKey;
     }
 
-    /**
-     * {@inheritDoc}
```

Comment:
Why are we removing all of these comments?  Is it because the java doc comments do not add any real information?

Reply:
Technically they don't help, but TBH, I love having them in anyway because it feels less "squishy" :-)

Reply:
Added them back. 😁